examples/convert_db_to_extxyz.py

```diff
@@ -0,0 +1,28 @@
+import os
+import sys
+from pathlib import Path
+
+from hippynn.databases.xyz_database_tools import load_base_database, write_extxyz
+
+def main():
+    if len(sys.argv) < 2:
```

Comment:
I would strongly prefer that we use argparse, it is easy to set up and contains a number of quality of life features with very little work. (such as `python script.py --help`)